src/main/java/javaslang/collection/HashSet.java

```diff
@@ -800,7 +800,7 @@ public boolean equals(Object o) {
 
     @Override
     public String toString() {
-        return mkString(", ", "HashSet(", ")");
+        return mkString("HashSet(", ", ", ")");
```

Comment:
Oh, thanks for catching - I thought I've catched all after changing `mkString(infix, prefix, suffix)` to `mkString(prefix, infix, suffix)`.